README.md

```diff
@@ -48,6 +48,10 @@ Toast container options (optional)
 [position]="'top-left' | 'top-right' | 'bottom-right' | 'bottom-left'"  (default: 'top-right')
 [transition]="'bounce' | 'slide' | 'zoom' | 'flip'" (default: 'bounce)
 [autoClose]="time in ms (0 = disabled)" (default: 5000)
+[autoSuccess]="time in ms (0 = disabled)" (default: undefined)
+[autoInfo]="time in ms (0 = disabled)" (default: undefined)
+[autoWarn]="time in ms (0 = disabled)" (default: undefined)
+[autoError]="time in ms (0 = disabled)" (default: undefined)
```

Comment:
Property names in the manual don't seem to match the ones in the implementation. (e.g. autoCloseSuccess, but manual states autoSuccess).